### DISCUSSION TITLE
Cannot compile error, builder error: folder

### DISCUSSION THREAD

**Discussion by AngelGamer29JA** [vlang/v#17478] [Questions and Answers]
I am having trouble installing and using Vlang on Linux Lite 6.2. After installing Vlang, when I try to test it using the terminal, I get the following error:

cannot compile /usr/local/v/cmd/tools/vrepl.v:
builder error: folder /usr/local/v/cmd/tools is not writable; code: 3

When I try to run "v init," I get the following error:

cannot compile /usr/local/v/cmd/tools/vcreate:
builder error: folder /usr/local/v/cmd/tools/vcreate is not writable; code: 3

**Comment by JalonSolov** [comment]
V needs to be installed in a directory that is writeable by your user id.  For example, many of the tools only get compiled the first time you run them.

Make sure you did NOT install V as root user, unless that is your regular user id on that system.  The only time even sudo is needed is when running `v symlink`.
